buildSrc/build.gradle.kts

```diff
@@ -1,10 +1,11 @@
 plugins {
-    kotlin("jvm") version "1.7.22"
+    `kotlin-dsl`
 }
 
 kotlin {
     jvmToolchain {
-        (this as JavaToolchainSpec).languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(8))
+        vendor.set(JvmVendorSpec.AZUL)
```

Comment:
It might be a good idea to include a comment in the commit message of the commit that adds this as to why Azul is being used. Or failing that, it would be good to add a comment here. This is the sort of detail that I could see the rationale being forgotten over time.